drivers/filesystems/fastfat_new/fatstruc.h

```diff
@@ -1371,7 +1371,8 @@ typedef struct _CCB {
     //  since it is used so often.  Line these up on byte boundaries for grins.
     //
 
-    ULONG Flags:24;
+    ULONG Flags; //  :24; // Due to https://developercommunity.visualstudio.com/t/Broken-runtime-checks-with-CL-19293013/1503629
+    // Note: this will not be packed anyway!
```

Comment:
This either needs `#ifdef __REACTOS__` or to be sent upstream.

Reply:
Maybe that by sending upstream they will fix it (either in fastfat or in the compiler side) quicker 🤔

Reply:
The code was already hacked, when it was imported.

Reply:
Ok, fixed